client/blocks/disconnect-jetpack/index.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import Dialog from 'components/dialog';
+import Button from 'components/button';
+
+class DisconnectJetpack extends Component {
+
+	planFeatures() {
+		const { plan, translate } = this.props;
+		const features = [];
+
+		switch ( plan ) {
+
+			case 'free':
+				features.push( translate( 'Site stats, related content, and sharing tools' ) );
+				features.push( translate( 'Brute force attack protection and uptime monitoring' ) );
+				features.push( translate( 'Unlimited, high-speed image hosting' ) );
+				break;
+
+			case 'personal':
+				features.push( translate( 'Daily, automated backups (unlimited storage)' ) );
+				features.push( translate( 'Priority WordPress and security support' ) );
+				features.push( translate( 'Spam filtering' ) );
+				break;
+
+			case 'premium':
+				features.push( translate( 'Daily, automated backups (unlimited storage)' ) );
+				features.push( translate( 'Daily, automated malware scanning' ) );
+				features.push( translate( 'Priority WordPress and security support' ) );
+				features.push( translate( '13Gb of high-speed video hosting' ) );
+				break;
+
+			case 'professional':
+				features.push( translate( 'Daily, automated backups (unlimited storage)' ) );
+				features.push( translate( 'Daily, automated malware scanning with automated resolution' ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Daily, automated malware scanning' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).